### PR TITLE
[NET 6 Attribute Conversion] Homekit

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

File: src/HomeKit/HMHome.cs [hunk 1]
```diff
@@ -87,57 +90,40 @@ public partial class HMHome
 
 #if !XAMCORE_4_0
 
-#if !NET
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
+		[SupportedOSPlatform ("maccatalyst14.0")]
+		[UnsupportedOSPlatform ("tvos")]
```
Comment (tj-devel709) [inline src/HomeKit/HMHome.cs:96]: I see the message for obsolete ios 9, but I don't see the Unsupported for it

File: src/HomeKit/HMHome.cs [hunk 2]
```diff
@@ -87,57 +90,40 @@ public partial class HMHome
...
-#endif // !NET
 		public virtual void RemoveUser (HMUser user, Action<NSError> completion) {
 			throw new NotSupportedException ();
 		}
 
-#if !NET
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
+		[SupportedOSPlatform ("maccatalyst14.0")]
+		[UnsupportedOSPlatform ("tvos")]
```
Comment (tj-devel709) [inline src/HomeKit/HMHome.cs:115]: Same

**Comment by chamons** [comment]
Test failure - https://github.com/xamarin/maccore/issues/2414

File: src/HomeKit/HMHome.cs (hunk 1 above)
Comment (chamons) [reply src/HomeKit/HMHome.cs:96]: Line 103. The diff has it broken up

Comment (chamons) [reply src/HomeKit/HMHome.cs:96]: Wait, actually, what is going on?

**Comment by chamons** [comment]
Merging main (and fixing the conflict) removed the cases TJ had concerns about.

**Comment by chamons** [comment]
Looks like networking failure, kicking again.

**Pull request merged**